seatunnel-e2e/seatunnel-connector-v2-e2e/connector-doris-e2e/src/test/java/org/apache/seatunnel/e2e/connector/doris/AbstractDorisIT.java

```diff
@@ -123,25 +123,47 @@ protected void initializeJdbcConnection()
     // The Host of the official image [apache/doris:doris-all-in-one-2.1.0] BE is 127.0.0.1, causing
     // cross-container access failure. Delete the BE and add it again
     private void initializeBE() {
-        try (Statement statement = jdbcConnection.createStatement()) {
-            ResultSet beResultSet = statement.executeQuery(SHOW_BE);
-            List<String> beList = new ArrayList<>();
-            while (beResultSet.next()) {
-                beList.add(beResultSet.getString("Host"));
-            }
-            if (beList.stream().anyMatch("127.0.0.1"::equals)) {
-                ResultSet resultSet = statement.executeQuery(SHOW_FE);
-                String feIp = null;
-                while (resultSet.next()) {
-                    feIp = resultSet.getString("Host");
+        int maxRetries = 5;
+        int retryIntervalSeconds = 10;
+        int attempt = 0;
```

Comment:
Hardcode this value is not a good option, we can use parameter with default value to control this.